README.md

```diff
@@ -620,12 +620,12 @@ The gem is available as open source under the terms of the [MIT License](https:/
 
 ## Code of Conduct
 
-Everyone interacting in the Xpect project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/[USERNAME]/xpect/blob/master/CODE_OF_CONDUCT.md).
+Everyone interacting in the Xpect project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/xpect/blob/master/CODE_OF_CONDUCT.md).
 
```

Comment:
The first pull request! 💯 ✨ . 

The link is missing the username; it should be `https://github.com/tdouce/xpect/blob/master/CODE_OF_CONDUCT.md`.